pyproject.toml

```diff
@@ -28,6 +28,33 @@ exclude = """(?x)(
     | ^tests/balance_xdist_plugin\\.py$         # not part of our test suite.
     )"""
 
+[tool.pytest.ini_options]
+minversion = "6.0"
```

Comment:
What does this do? It wasn't in the setup.cfg.

Reply:
https://docs.pytest.org/en/7.1.x/reference/customize.html#pyproject-toml

Reply:
I'm not sure we need this setting, since we pin the version of pytest.  I think it would be better to leave it out.

Reply:
Yeah, when I migrated our version, I included it to be on the safe side, but I agree this is likely ok to leave out.